.cirrus.yml

```diff
@@ -0,0 +1,63 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
+fedora_30_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-fedora-30
+  env:
+    PYTHON: python3
+    matrix:
+      - AVOCADO_SRC: avocado-framework<70.0
+      - AVOCADO_SRC: -e git+https://github.com/avocado-framework/avocado#egg=avocado_framework
+    matrix:
+      - SETUP: setup.py develop --user
+      - SETUP: -m pip install .
+  setup_script: &setup_scr
+    - $PYTHON --version
+    - $PYTHON -m pip install $AVOCADO_SRC
+    - $PYTHON -m pip install -r requirements.txt
+    - $PYTHON $SETUP
+  bootstrap_script: &bootstrap_scr
+    - $PYTHON -m avocado vt-bootstrap --vt-skip-verify-download-assets --yes-to-all
+  list_script: &list_scr
+    - $PYTHON -m avocado list
+  dry_run_script: &dry_run_scr
+    - $PYTHON -m avocado --show all run --dry-run -- boot
+
+rhel_8_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-rhel-8
+  env:
+    PYTHON: python3
+    matrix:
+      - AVOCADO_SRC: avocado-framework<70.0
+      # When installing from source, pip tries to install at
+      # /usr/local/lib/python3.6/site-packages/ and fails with:
+      # [Errno 2] No such file or directory: '/usr/local/lib/python3.6/site-packages/test-easy-install-78.write-test'
+      # workaround is to add "--prefix /"
+      - AVOCADO_SRC: --prefix / -e git+https://github.com/avocado-framework/avocado#egg=avocado_framework
```

Comment:
Hmm, interesting. Did you have enough time to investigate it? Is it their bug, or ours? RHEL8 only or py3? (just wondering)

Reply:
(I mean apparently it works well in Fedora...)